apps/api/src/config/index.ts

```diff
@@ -139,6 +139,8 @@ const parseEvmChains = (): SpecificChain[] => {
 export const config = {
   server: {
     port: parseInt(process.env.PORT || "3000", 10),
+    // TODO: these ports are going to be put into the openapi json spec, so they can't really be set at runtime, wtd?
+    testPort: 3001,
```

Comment:
Since the spec defines the port, we can't set it based on an env at runtime.  There's a few options here, so open to ideas and discussion.  I'm thinking we find a resolution to this in the PR to update all the tests to use the SDK.